lms/views/decorators/h_api.py

```diff
@@ -11,10 +11,10 @@
 
 def create_h_user(wrapped):  # noqa: MC0001
```

Comment:
Is this function name still appropriate?

Reply:
Rename to `create_or_update_h_user`?

Reply:
It's actually more like `update_or_create` I guess. For the group we've already called the same thing `upsert_group()` so maybe `upsert_user()`? I know it's calling `PATCH` and `POST` APIs not `PUT`, but it is effectively an upsert: it tries to do an update and if that fails it does a create. It's just that the upsert logic is in the lms app itself not in h.